home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/Address.java

```diff
@@ -0,0 +1,17 @@
+package com.softserveinc.ita.homeproject.homedata.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Address extends BaseEntity {
```

Comment:
According to the design spec Address contains a bit more fields. How are you going to store it if you didn't specify it in Entity?